crates/polars-core/src/chunked_array/logical/duration.rs

```diff
@@ -11,6 +11,26 @@ impl Int64Chunked {
     }
 }
 
+impl Float64Chunked {
```

Comment:
`into_duration` make no sense for floating point arrays.

You should cast somewhere else and then call `into_duration`.